src/components/driftsmelding/DriftsmeldingUtilities.ts

```diff
@@ -36,16 +36,16 @@ export type Driftsmelding
 
 export const getDriftsmeldingByKommuneResponse = (kommuneResponse: KommuneResponse | undefined) => {
     if (kommuneResponse) {
-        if ((
-            kommuneResponse.erInnsynDeaktivert || kommuneResponse.erInnsynMidlertidigDeaktivert) &&
-            (kommuneResponse.erInnsendingEttersendelseDeaktivert || kommuneResponse.erInnsendingEttersendelseMidlertidigDeaktivert)
+        if (kommuneResponse.erInnsynMidlertidigDeaktivert &&
```

Comment:
Bare litt småpirk, men jeg ville kanskje ha innført en variabel som forklarer hva det testes på:

```
const ettersendelseMidlertidigDeaktivert: boolean = kommuneResponse.erInnsynMidlertidigDeaktivert && kommuneResponse.erInnsendingEttersendelseDeaktivert && kommuneResponse.erInnsendingEttersendelseMidlertidigDeaktivert
```

Reply:
Jeg gjorde en kjapp test, og syns ikke lesbarheten ble bedre, så jeg lar det være for nå.